hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/ContainerProtocolCalls.java

```diff
@@ -578,4 +581,35 @@ private static Text getService(DatanodeBlockID blockId) {
     validators.add(validator);
     return validators;
   }
+
+  public static HashMap<DatanodeDetails, GetBlockResponseProto>
+      getBlockFromAllNodes(
```

Comment:
This new introduced method is not used here, what's this method intended used for?

Reply:
not sure, why these changes are showed up in this PR. These changes were not made as part of this PR.
I have re-uploaded the changes and it should go away.